main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = "shush"
-	app.Version = "1.4.1"
+	app.Version = "1.4.1-with-new-aws-sdk"
```

Comment:
Maybe this is pedantry, but i'd almost argue that bringing in the updated SDK unlocks new functionality (i.e., understanding EKS's secret token store), so perhaps at least a minor version bump would be in order?  Or perhaps this is something the maintainers are expected to take care of when merging patches anyway 🤔

Reply:
Oh, whoops >_>